package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-device-manager",
-  "version": "0.4.14",
+  "version": "0.4.16",
```

Comment:
This should be done automatically by the release tool

Reply:
done